flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/catalog/FunctionLookup.java

```diff
@@ -33,6 +33,10 @@
 
 	/**
 	 * Lookup a function by name. The lookup is case insensitive.
+	 *
+	 * TODO: 1) replace string with ObjectIdentifier so users can reference functions across catalogs
+	 * TODO: 2) should pass in an array of Expressions to get literals and input arg types, in order
+	 * 	to instantiate Hive functions, compute the result/accumulator type, and create corresponding FunctionDefinition
```

Comment:
The list of literals/input arg types should not be part of the lookup as they are not known at this point.

btw: can we remove all these TODOs from the PR. TODO's make sense if one functionality is missing in a class. But the function lookup is under heavy refactoring and we could add a lot of TODOs everywhere that would make the code unreadable.